docs/data/_index.md

```diff
@@ -13,7 +13,8 @@ aliases:
   - "/data-management/"
   - "/data-management/"
   - "/services/data/"
-icon: "/services/icons/data-management.svg"
+icon: true
+images: ["/services/icons/data-management.svg"]
 imageAlt: "The data management service."
```

Comment:
Lots of these have sorta meaningless imageAlt...do you want to delete it too in this PR?